template.go

```diff
@@ -73,9 +73,9 @@ OPTIONS:
    {{end}}{{end}}
 `
 
-var MarkdownDocTemplate = `% {{ .App.Name }}(8) {{ .App.Description }}
-
-% {{ .App.Author }}
+var MarkdownDocTemplate = `% {{ .App.Name }}(8){{ if .App.Description }} {{ .App.Description }}{{ end }}
+{{ range $a := .App.Authors }}
+% {{ $a }}{{ end }}
```

Comment:
Whst if authors is not set just author, that would be confusing as it would not work.

Reply:
I thought this will catch it:
https://github.com/urfave/cli/blob/63cd2e3d6bb5c78854dd1d5d93c5a46782c1f36c/app.go#L142-L144

Reply:
Ah yes.